### DISCUSSION TITLE
Help subsections in hydra config file

### DISCUSSION THREAD

**Discussion by o-fares** [facebookresearch/hydra#2540] [Q&A]
Hello,

I am currently using the hydra config file as a part of a deep learning framework, in order to declare several parameters.
I wrote a help section in my config.yaml file to declare all the parameters and describe them:

```
help:
    header: == HYDRA CONFIG FILE ARGUMENTS DESCRIPTION ==


    template: |-
      ${hydra.help.header}
      The hydra config file is used to fill in the arguments for the segmentation monai framework.
      It is split in several parts with arguments that will be used for the training:

      - data
        - root_path:            TYPE=STR
                                root directory containing the data and gt folder that contains files in nifti format.
                                Directory must also contains the db.csv dataframe that list the patients IDs and TAGS
        ...

     - split
        - run:                  TYPE=BOOL
                                Whether to run the split part or not.
        - eval_ratio:           TYPE=FLOAT
                                Split ratio for the eval.
        - test_ratio:           TYPE=FLOAT
                                Split ratio for the test.
        ...

     - train:
        - run:                  TYPE=BOOL
                                Whether to run the train part or not.
        - seed:                 TYPE=INT
                                Define a seed if you want to run deterministic training for reproductibility
        ...

```

I know that the command to display the help is the following one:
```python d:/projects/my_file.py --help```

The problem is that this command displays the full help section. My question is: **is there a way to display only a subsection of this help message, only the "data" part for example ?** 

Thank you for your help.

**Comment by Jasha10** [comment]
Currently (2022-01-11) there is no smooth way to access just a sub-part of the help via the CLI. There is an open feature https://github.com/facebookresearch/hydra/issues/633 that's related to this topic.

**Reply by Jasha10** [reply]
Cross link to related stackoverflow question: https://stackoverflow.com/questions/75081038/how-can-i-access-to-a-help-subsection-from-a-hydra-config-file-in-a-command-line/75081851#75081851

**Comment by omry** [comment]
I consider this a duplicate of adding proper support for per field info in the help.
